### ISSUE TITLE
Videomass 5.0.16 - Strings untranslated

### ISSUE THREAD

**Issue by bovirus** [jeanslack/Videomass#339]
## Thanks for taking the time to submit a bug report! Please include the following information:   

**What operating system are you using?**   
Windows 10 22H2 64bit italian

**What version of Videomass are you using and how did you install or run it?**   
5.0.16

**Describe the bug**   
Using italian language file (my file updated) some windows seems have untranslated strings
Please check screenshots.

![Videomass 5,0 16 - GUI - 01](https://github.com/jeanslack/Videomass/assets/1262554/49281106-9a62-4b61-960e-ad09ac1164a6)

![Videomass 5,0 16 - GUI - 02](https://github.com/jeanslack/Videomass/assets/1262554/12c5efea-4c22-47fd-8f7f-6c83b2691c12)

![Videomass 5,0 16 - GUI - 03](https://github.com/jeanslack/Videomass/assets/1262554/a6c02eaa-58b6-4c59-9e9f-93c24c6fff96)

![Videomass 5,0 16 - GUI - 04](https://github.com/jeanslack/Videomass/assets/1262554/d989180b-d0fe-4ee2-be3a-f55c29645f74)

![Videomass 5,0 16 - GUI - 05](https://github.com/jeanslack/Videomass/assets/1262554/0c6d537e-ec2c-456c-b86b-0965cb740e2d)

**Comment by jeanslack** [comment]
Thank you so much for reporting this issue.

I'm currently investigating the reasons for not translating strings on stock buttons on MS-Windows. Those [stock buttons](https://docs.wxpython.org/stock_items.html) should be automatically translated into the default language and do not need to be translated manually.

Try this workaround: set the Italian language in the Preferences dialog and that problem should partially resolve itself.
Let me know.

**Comment by bovirus** [comment]
@jeanslack

The GUi is already ste in Italian and screenshot are get with altest italian language.

Please chek PR here 
https://github.com/jeanslack/Videomass/pull/341

**Comment by jeanslack** [comment]
Yes, I saw, it is probably set to `System default language`, you try setting the language specifically to `Italian` from the drop-down menu.

**Comment by bovirus** [comment]
I set it as Italian but many buttons remian in English (Help/Close/etc).

**Comment by jeanslack** [comment]
I just tried today, by setting the Preferences to the Italian language the program is translated correctly.

![Immagine 2024-07-06 170400](https://github.com/jeanslack/Videomass/assets/8676484/e538e4d0-ce2b-4c4c-ac42-46e2adc94c10)
![Immagine 2024-07-06 170223](https://github.com/jeanslack/Videomass/assets/8676484/ae7db0bc-f4f9-49e0-a93c-1d7c59f9a8a3)

**Comment by bovirus** [comment]
@jeanslack 

Please check.
I set the Italian and restart the application.

![image](https://github.com/jeanslack/Videomass/assets/1262554/4fef6161-211c-422f-ab47-8038a509e041)

![image](https://github.com/jeanslack/Videomass/assets/1262554/0f54bc26-4ce3-4da3-bff8-b7c8558458d9)

![image](https://github.com/jeanslack/Videomass/assets/1262554/24836359-c5ba-4ba8-8a52-3980503cfe90)

![image](https://github.com/jeanslack/Videomass/assets/1262554/0a39511b-68cd-43e3-9c9f-5dd6b87b63de)

**Comment by jeanslack** [comment]
Can you please specify which videomass package you have installed and are using?
Note that I am not referring to the version but the type of package/installation.

**Comment by bovirus** [comment]
5.0.16 portable version.

https://github.com/jeanslack/Videomass/releases/download/v5.0.16/Videomass-v5.0.16_x86_64-portable.7z

My configuration is Windows 10 22H2 64bit Italian.

**Comment by jeanslack** [comment]
Thanks for the info.
In fact I have just noticed that the portable version has this problem. 
I had tested starting the application from source and also the one installed with pip and both did not present any problems.

The portable version for Windows is built with [pyinstaller](https://github.com/pyinstaller/pyinstaller).
I will evaluate what to do as soon as I report the problem to the pyinstaller developers.
